openforcefield/tests/test_toolkits.py

```diff
@@ -1778,11 +1703,10 @@ def test_substructure_search_on_large_molecule(self):
 
 
 
+@requires_rdkit
```

Comment:
We should make another decorator for `@requires_ambertools` here, which could ensure that `which antechamber` resolves successfully.